migrations/deprecated/00013_targeted_messaging_add_outreach_2/index.sql

```diff
@@ -0,0 +1,21 @@
+INSERT INTO outreaches (
+    source_id,
+    name,
+    start_date,
+    end_date,
+    type,
+    team_name,
+    source_file,
+    source_file_checksum,
+	target_all
+) VALUES (
+    2,
+    'micro-campaign',
+    '1970-01-01 00:00:00.000',
+    '1970-01-01 00:00:00.000',
+    '',
+    '',
+    '',
+    '',
```

Comment:
Is there a reason to not populate these values? Afaiu, we need to add the checksum to target.

Reply:
@iamacook I think it's because this iteration does not target specific safes and therefor we don't have any file and checksum to validate.

Reply:
If that's the case, I'd exclude this migration.

Reply:
In this case we don't have a file with the targeted safes since we are showing to all safes. And the checksum was for the targeted safes file

Reply:
If the primary goal of this PR is functionality, I’d recommend splitting the creation of the outreach into a separate PR. This would keep the focus narrower and the diff. smaller. For example, if the migration were to fail, it would be easier to pinpoint whether the functionality or migration caused the issue.

Regarding the values: what are your thoughts on making `source_file` and `source_file_checksum` nullable instead of setting them to an empty string? This approach might better align with their intended semantics.

I’d also suggest using more specific dates where possible for clarity.

Reply:
I removed the outreach creation, will do in another PR